app/lib/frontend/templates/views/pagination.mustache

```diff
@@ -5,8 +5,8 @@
 <ul class="pagination">
   {{#page_links}}
     <li class="{{#active}}-active{{/active}}{{#disabled}}-disabled{{/disabled}}">
-      <a href="{{#render_link}}{{href}}{{/render_link}}"{{#rel_prev}} rel="prev"{{/rel_prev}}{{#rel_next}} rel="next"{{/rel_next}}>
-        <span>{{{text}}}</span>
+      <a href="{{#render_link}}{{& href}}{{/render_link}}"{{#rel_prev}} rel="prev"{{/rel_prev}}{{#rel_next}} rel="next"{{/rel_next}}>
+        <span>{{& text}}</span>
```

Comment:
Why do we have custom escaping here?

Reply:
`{{{ }}}` is the same as `{{& }}` == non-escaped content. I've changed the three `{` to the other version because I think it is easier to notice what's happening there.